utils/test_summary.py

```
@@ -80,25 +71,25 @@ def print_summary(json_file="test-results/report.json"):
-            red(stats["failed"]) if stats["failed"] else "0",
+            stats["passed"],
+            stats["failed"],
             stats["skipped"],
             stats["other"],
         ])
 
     # --- Add final summary row ---
     if total_failed == 0:
-        summary_icon = green("✓")
-        summary_text = bold(green(f"{total_passed}/{total_tests} tests passed"))
+        summary_label = f"✓ ALL PASSED"
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)